subsys/bluetooth/host/gatt.c

```diff
@@ -35,7 +35,8 @@
 #include "settings.h"
 #include "gatt_internal.h"
 
-#define SC_TIMEOUT K_MSEC(10)
+#define SC_TIMEOUT	K_MSEC(10)
```

Comment:
They are aligned, it's only diff's "+" messing this

Reply:
What messes things up here is that you use a tab on this line but a space on the one below it. Please be consistent with that. In this case it should be spaces - tabs should only be used for indentation (not alignment)